carbon-design-system/carbon
# [DRAFT] feat(Disclosure): added initial disclosure component and tests

Closes #6633 


- This is a draft PR for the base of the disclosure component 
- Next steps: 

- [ ] add some basic styles
   - testing for onClick 
   - make sure ID stays on node 
   - make sure REF stays on node
